tools/clang/test/CMakeLists.txt

```diff
@@ -77,7 +77,11 @@ endif()
 set(CLANG_TEST_DEPS dxc dxa dxopt dxl dxv dxr dxcompiler clang-tblgen llvm-config opt FileCheck count not ClangUnitTests)
 if (WIN32)
 list(APPEND CLANG_TEST_DEPS
-     dxc_batch ExecHLSLTests
+     dxc_batch ExecHLSLTests dxildll
+    )
+elseif(NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
```

Comment:
I'm confused by this. Why is this needed?

Reply:
Shall we enable libdxil to mac?
We only release dxildll for linux and windows now.

Reply:
I think one of the biggest benefits of us open sourcing this is to allow people to build it on macOS where we don't currently have supported binary releases.